docs/tempo/website/operations/_index.md

```diff
@@ -1,10 +1,11 @@
 ---
-title: Operations
+title: Deployment and Operations
 weight: 300
 ---
 
-Operations for Tempo include
+- [Deployment](deployment)
 - [Monitoring](monitoring)
 - [Tempo CLI](tempo_cli)
+- [Architecture](architecture)
```

Comment:
I liked the idea of having Architecture on the top layer (like a page, we don't need a folder for it). 

No strong feelings about putting this inside Operations, just commenting on it :smile: